components/templates/NewsPage/Contacts.js

```diff
@@ -10,73 +9,71 @@ export default function Contacts({ contacts }) {
   return (
     <>
       <Styled.ArticleHeading>{t(`news.contacts`)}</Styled.ArticleHeading>
-      <div>
+      <Styled.ContactList>
```

Comment:
quick re-work of the markup here to make an unordered list of `address` elements for each contact